thefuck/rules/git_lfs_mistype.py

```diff
@@ -0,0 +1,18 @@
+import re
+from thefuck.utils import get_all_matched_commands, replace_command
+from thefuck.specific.git import git_support
+
+
+@git_support
+def match(command):
+    '''
+    Match a mistyped command
+    '''
+    return 'lfs' in command.script and 'Did you mean this?' in command.output
+
+
+@git_support
+def get_new_command(command):
+    broken_cmd = re.findall(r'Error: unknown command "([^"]*)" for "git-lfs"', command.output)[0]
+    matched = get_all_matched_commands(command.output, ['Did you mean'])
```

Comment:
If you place a breakpoint right after line 17 (e.g `__import__("ipdb").set_trace()` after installing ipdb with `pip install ipdb`) you can see that `matched` contains some undesired content:

```
ipdb> l
     16 def get_new_command(command):
     17     broken_cmd = re.findall(r'Error: unknown command "([^"]+)" for "git-lfs"',
     18                             command.output)[0]
     19     matched = get_all_matched_commands(command.output, 'Did you mean')
     20     __import__("ipdb").set_trace()  # FIXME: BREAKPOINT!!!
---> 21     return replace_command(command, broken_cmd, matched)

ipdb> matched
['env', 'ext', "Run 'git-lfs --help' for usage."]
```

To get the debugger taking input, I ran TheFuck directly, instead of via `fuck`, like this: `thefuck git lfs evn`

Checking `get_all_matched_commands` source code:

https://github.com/nvbn/thefuck/blob/3c542a5b8cde085126fd9197b1ca6967c7adab1e/thefuck/utils.py#L148-L159

You can see that it supports `separator` being a list of strings. So, how about adding `' for usage.'` to the list of separators passed to `get_all_matched_commands`? This way, `matched` will be equal to `['env', 'ext']`.

Reply:
Thank you for catching that! I didn't even think about looking at that.

I've made the requested change and it no longer contains the unwanted string.